callisto/delivery/hashers.py

```diff
@@ -0,0 +1,231 @@
+import base64
```

Comment:
Can you go into more detail about why subclassing would not be possible/why we want to remove the dynamic features? I very strongly prefer using Django's code to the extent we can--one so that we can piggyback on their security patches, and two because of licensing murkiness.